org.eclipse.dawnsci.analysis.api/src/org/eclipse/dawnsci/analysis/api/persistence/IPersistentFile.java

```diff
@@ -8,6 +8,8 @@
  */
 package org.eclipse.dawnsci.analysis.api.persistence;
 
+import java.util.Arrays;
+import java.util.Collection;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Collection'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)